stackslib/src/chainstate/nakamoto/mod.rs

```diff
@@ -130,7 +130,7 @@ pub use self::staging_blocks::{
     NakamotoStagingBlocksConn, NakamotoStagingBlocksConnRef, NakamotoStagingBlocksTx,
 };
 
-pub const NAKAMOTO_BLOCK_VERSION: u8 = 0;
+pub const NAKAMOTO_BLOCK_VERSION: u8 = 1;
```

Comment:
Wait, I don't think we want this change, right? This would fail block replay.

Reply:
I'm also not sure why this is useful, other than in the `post_blockproposal` check.

Reply:
Gotcha - so there's no issue of old nodes suddenly seeing blocks with a different version? Looking at the code, it doesn't look like it, but I just wanted to double check.

Reply:
Nope, no issue at all.  Block-processing doesn't consider the block version (except for shadow blocks, but even then it only looks at the high bit).